packages/extension-browser/src/devtools/views/pages/results/problem.tsx

```diff
@@ -24,7 +24,7 @@ const Problem = ({ problem, index }: Props) => {
     const onViewSourceClick = useCallback((event: MouseEvent) => {
         if (browser.devtools.panels.openResource) {
             event.preventDefault();
-            browser.devtools.panels.openResource(problem.resource, line, () => {});
+            browser.devtools.panels.openResource(problem.resource, line, () => { });
```

Comment:
Let's avoid formatting-only changes in files (I suspect one of your tools did this automatically)